components/ui/common/NoResultView.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Text, useTheme, VStack } from "native-base";
+import { InterfaceVStackProps } from "native-base/lib/typescript/components/primitives/Stack/VStack";
+
+export interface INoResultViewProps extends InterfaceVStackProps {
+  message?: string;
+}
+
+function NoResultView({ message, ...rest }: INoResultViewProps) {
+  const theme = useTheme();
+
+  return (
+    <VStack flex={1} justifyContent="center" alignItems="center" {...rest}>
```

Comment:
I think instead we could give this a `type` that would be like `comment` or `post` and return a different message based on that. And if no type is provided we just return a default message "No results found :("

Reply:
@sgriff96 Agreed, and fixed